Sprite atlas container and processing with tags

silfaber opened the pull request:
warning: does not actually contain a charactermanager
using #speaker:A/B and #emotion:happy/sad/neutral/angry will prompt emotions to change and dictate which name shows up above the text box when the dedicated character is speaking

silfaber committed:
support for speaker and emotion tags

current structure: '#speaker:A/B #emotion:angry/sad/happy/neutral' written after each line where they change
added sprite atlas for testing
turned up max size of background assets
Todo:
Add name on top of dialogue box when speaking
add dialogue state for after player picks choice

silfaber committed:
player dialogue display and speaker name

implemented speaker box on top of dialogue
made player's chosen dialogue show up after choice so emotion and name are displayed
fixed bug where tags check call still occurs after end of dialogue

silfaber committed:
removed character manager and cleaned up code

didn't end up using charactermanager in favor of adding sprite atlases to dialogue scriptable objects

silfaber committed:
scene update

i will never remember to save the scene before making a pull request

JdeHaan2001 commented on Assets/Scripts/Dialogue/InkManager.cs:
```diff
@@ -113,13 +108,10 @@ private void InstantiateChoiceButtons()
 
     private void HandleTextTags(string pText)
     {
+        string speakerID = "B";
```
Could you make it more clear what "B" is. Maybe rename it to "Character_B" or something

JdeHaan2001 commented on Assets/Scripts/Dialogue/InkManager.cs:
```diff
@@ -173,17 +169,56 @@ private string GetSpeakerTagValue(string pTag)
     {
-        playerImage.sprite = dialogue.CharacterASprite;
-        npcImage.sprite = dialogue.CharacterBSprite;
+        if (pSpeakerID == "A")
```
Can you make "A" a const variable? Then it's easier to change the value if necessary.

JdeHaan2001 commented on Assets/Scripts/Dialogue/InkManager.cs:
```diff
@@ -173,17 +169,56 @@ private string GetSpeakerTagValue(string pTag)
+        {
+            nameText.text = dialogue.CharacterAName;
+        }
+        else if (pSpeakerID == "B")
```
Same as before, can you make "B" a const variable?

JdeHaan2001 reviewed (changes requested):
Overall looks good, but check comments

silfaber committed:
requested changes

changed magic variables A and B to a const string property for player and NPC respectively

silfaber committed:
small additional cleanup

JdeHaan2001 reviewed (approved):
Looks okiedokie
